### PR TITLE
Migrate docs to docs.scrt.network

### PR DESCRIPTION
**Pull request by dylanschultzie**
https://github.com/SecretFoundation/docs/pull/28

Once this is merged, build.scrt.network will be brought down and all content will reside at: https://github.com/SecretFoundation/docs

### CODE REVIEW

**Comment by dylanschultzie** [comment]
Current build of docs.scrt.network can be seen here: https://b9ad4af8.secretdocs.pages.dev/

**Comment by Cashmaney** [comment]
@dylanschultzie this is fine with me. Can we merge this whenever, or should we sync first to elegantly bring the old docs page down?

**Comment by moonstash** [comment]
> [SecretFoundation/docs#28](https://github.com/SecretFoundation/docs/pull/28)
> 
> Once this is merged, build.scrt.network will be brought down and all content will reside at: https://github.com/SecretFoundation/docs

I went ahead and merged it just now.

**Comment by dylanschultzie** [comment]
> @dylanschultzie this is fine with me. Can we merge this whenever, or should we sync first to elegantly bring the old docs page down?

I can be merged now - this doesn't actually have an effect on build.scrt.network since we don't know how that's being used anymore.

**Comment by Cashmaney** [comment]
Aight, bye bye old docs!

**Comment by assafmo** [comment]
@dylanschultzie can you also fix the links in README.md that are now became broken, please? Basically all the relative links.

File: docs/lcd-server-example.md [hunk 1]
```diff
@@ -1,112 +0,0 @@
-1. Put this file in `/etc/systemd/system/secret-lcd.service`
```
Comment (assafmo) [inline docs/lcd-server-example.md:1]: @dylanschultzie where did this file go? I can't find it on docs.scrt.network

Comment (dylanschultzie) [reply docs/lcd-server-example.md:1]: @assafmo https://github.com/SecretFoundation/docs/blob/main/docs/lcd-server-example.md

For docs that didn't have pages on build.scrt.network, I left them out on docs.scrt.network as well

**Pull request merged**